packages/pwa-kit-react-sdk/src/ssr/server/express.js

```diff
@@ -258,13 +265,21 @@ export const createApp = (options) => {
     }
 
     // Healthcheck
-    app.get('/mobify/ping', (_, res) =>
+    app.get(/^\/(pwa-kit|mobify)\/ping$/, (_, res) =>
         res
             .set('cache-control', NO_CACHE)
             .sendStatus(200)
             .end()
     )
 
+    // Configuration endpoint
+    app.get(/^\/(pwa-kit|mobify)\/config$/, (req, res) =>
+        res
+            .set('cache-control', NO_CACHE)
+            .json(req.config)
+            .end()
+    )
```

Comment:
If this endpoint isn't used by the code in this PR (I can't see that it is anywhere), then I would recommend not exposing it for two reasons:

1. It's extra stuff that were expected to support on an ongoing basis, without a use-case.
2. I believe people would be surprised to see that the content of a config file is immediately made available on a public API endpoint.

I know that it would get embedded in public code anyway, even without this endpoint, but do you think that exposing configs like this would be surprising from a security perspective?

Reply:
This was used for testing. I've since removed it.